packages/@aws-cdk/aws-codebuild/README.md

```diff
@@ -81,18 +77,17 @@ import s3 = require('@aws-cdk/aws-s3');
 
 const bucket = new s3.Bucket(this, 'MyBucket');
 new codebuild.Project(this, 'MyProject', {
-  source: new codebuild.S3BucketSource({
+  source: new codebuild.S3Source({
     bucket: bucket,
     path: 'path/to/file.zip',
   }),
 });
+// or: codebuild.Source.s3({
```

Comment:
Let's recommend the union-like approach instead of the free-floating classes. I would even argue that the classes should be internal. Not sure there is value in exposing them at all.

Reply:
Done.